### DISCUSSION TITLE
Different Arch, different names

### DISCUSSION THREAD

**Discussion by blackcrack** [doublecmd/doublecmd#595] [Suggestions]
Hi,

why not let renamed the binary's to an Achitecturebinarys and a starter before :  

\doublecmd\doublecmdx86_64.exe
\doublecmd\doublecmdx86.exe

and a standard starter who checks which architecture is existent and 
start the right exe-file.
\doublecmd\doublecmd-starter.exe

Therewith it is also possible to add the doublecmd-starter.exe to an Portable menu and 
supports the both most used Achitectures.

the Starter could give further the switches to the right binarys..

Therewith would it also possible to install once Plugin and use it with both Architectures.. 
in one Folder..
Also one doublecmd.xml and other configuration Files for both Architectures.

I download only the zip files.. by side ..

(thinking loud about.. it is also possible for add the linuxbinarys as portable directory ??????
so maybe a doublecmd-starter for linux binarys... therewith could it be possible also add 
doublecmdx86 and doublecmd86_64 for linux in the folder on the usb-stich ? )

best

**Comment by alexx2000** [comment]
Hi, because DC is not designed in this way and I don't want to change it.
